src/Controller/HomeController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use App\Entity\Event;
+
+class HomeController extends BaseController
+{
+    const ITEMS_ON_HOMEPAGE = 6;
```

Comment:
@daria-sieroshtan 
IMO better put it in base controller

Reply:
This is homepage-specific constant